index.js

```diff
@@ -11,15 +11,15 @@ module.exports = {
     var config = Config.load(this.project.root);
 
     if (config.buildError.notify) {
-      notifier.buildError(error);
+      notifier.buildError(error, config.buildError.options);
     }
   },
 
   postBuild: function(results) {
     var config = Config.load(this.project.root);
 
     if (config.buildSuccess.notify) {
-      notifier.buildSuccess(results);
+      notifier.buildSuccess(results, config.buildSuccess.options);
```

Comment:
Yes they should be! Good catch.

It's an awesome package, I've been looking for something like this for a while, and thought I would have to roll my own